web/book/src/reference/syntax/literals.md

```diff
@@ -167,7 +172,7 @@ derive first_check_in = start + 10days
 
 ### Examples
 
-Here's a fuller list of date and time examples:
+Here's a larger list of date and time examples:
```

Comment:
Yes better. 

Your grammar is better than mine!

Reply:
Probably not, but I have more time to spend on fussy stuff... :-)